ppml/trusted-big-data-ml/python/docker-gramine/base/start-scripts/start-python-fgboost-server-sgx.sh

```diff
@@ -15,9 +15,8 @@ do
 done
 cd /ppml/trusted-big-data-ml
 export sgx_command="/opt/jdk8/bin/java\
-        -cp '/ppml/trusted-big-data-ml/work/bigdl-${BIGDL_VERSION}/jars/bigdl-ppml-spark_${SPARK_VERSION}-${BIGDL_VERSION}.jar:/ppml/trusted-big-data-ml/work/spark-${SPARK_VERSION}/jars/commons-lang3-3.10.jar:/ppml/trusted-big-data-ml/work/bigdl-${BIGDL_VERSION}/jars/*:/ppml/trusted-big-data-ml/work/spark-${SPARK_VERSION}/conf/:/ppml/trusted-big-data-ml/work/spark-${SPARK_VERSION}/jars/*'\
+        -cp '/ppml/trusted-big-data-ml/work/bigdl-2.2.0-SNAPSHOT/jars/bigdl-ppml-spark_3.1.2-2.2.0-SNAPSHOT.jar:/ppml/trusted-big-data-ml/work/spark-3.1.2/jars/commons-lang3-3.10.jar:/ppml/trusted-big-data-ml/work/bigdl-2.2.0-SNAPSHOT/jars/*:/ppml/trusted-big-data-ml/work/spark-3.1.2/conf/:/ppml/trusted-big-data-ml/work/spark-3.1.2/jars/*:/ppml/trusted-big-data-ml/work/spark-3.1.2/examples/jars/*'\
```

Comment:
should use ${BIGDL_VERSION} ${SPARK_VERSION}
we've just changed spark to 3.1.3

Reply:
Fixed.